drawer_length: 0.27
dbc_params:
    perceive_handle:
        x_offset: -0.3
    open_drawer:
        x_offset: -0.62
    open_drawer_post:
        x_offset: -0.65
    open_drawer_post_safe:
        x_offset: -0.7
    perceive_drawer:
        x_offset: -0.30
    close_drawer_pre_safe:
        x_offset: -0.7
    close_drawer:
        x_offset: -0.6
    close_drawer_safe:
        x_offset: -0.75
